arex-storage-web-api/src/main/java/com/arextest/storage/beans/IndexesSettingConfiguration.java

```diff
@@ -22,7 +20,7 @@
 
 @Slf4j
 @Configuration(proxyBeanMethods = false)
-public class IndexsSettingConfiguration {
+public class IndexesSettingConfiguration {
```

Comment:
How to solve the update ttl index scenario？

Reply:
Need to add drop indexes logic.